db/migrate/20150326084125_create_customers.rb

```diff
@@ -0,0 +1,11 @@
+class CreateCustomers < ActiveRecord::Migration
+  def change
+    create_table :customers do |t|
+      t.string :name
+      t.references :organization, index: true
+
+      t.timestamps
+    end
+    add_index :customers, :name, unique: true
```

Comment:
@Gataniel I think it should be uniq for one organization only, so index should be for `[:name, :organization_id]`.